smoke-test/smoke.sh

```diff
@@ -20,6 +20,7 @@ pip install --upgrade pip wheel setuptools
 pip install -r requirements.txt
 
 echo "DATAHUB_VERSION = $DATAHUB_VERSION"
+export DATAHUB_TELEMETRY_ENABLED=false
 datahub docker quickstart --quickstart-compose-file ../docker/quickstart/docker-compose-without-neo4j.quickstart.yml --dump-logs-on-failure
```

Comment:
add it specifically to the invocation?
`DATAHUB_TELEMETRY_ENABLED=false datahub docker quickstart ...`

Reply:
Do we want telemetry if someone runs a local smoke test?